frontend/dockerfile/dockerfile_test.go

```diff
@@ -1028,17 +1028,27 @@ RUN e="300:400"; p="/file"                         ; a=` + "`" + `stat -c "%u:%g
 }
 
 func testCopyWildcardCache(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM busybox AS base
 COPY foo* files/
 RUN cat /dev/urandom | head -c 100 | sha256sum > unique
 COPY bar files/
 FROM scratch
 COPY --from=base unique /
-`)
+`, `
+FROM nanoserver AS base
+USER ContainerAdministrator
```

Comment:
nit: did you need `ContainerAdministrator` for this?

Reply:
apparently yes,  `RUN echo test> /unique` requires admin user